### DISCUSSION TITLE
IDropdownStyles not working, it gets overwritten

### DISCUSSION THREAD

**Discussion by Rafaelki** [microsoft/fluentui#16107] [Q&A] [answered]
### Environment Information

- **Package version(s)**: latest

### Please provide a reproduction of the bug in a codepen:

https://codepen.io/rafaelki/pen/KKgdPOj

#### Actual behavior:

The width of the callout passed in the "styles" property gets overwritten, so there is not way to set the width of the callout to 'auto' in order to display long options.

#### Expected behavior:

I would expect that the properties that I pass to the component take effect, overwritting the default behaviour.

### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: Normal

**Comment by JustSlone** [comment, marked as answer]
Actually with dropdown since there is coordination between the dropdown and the callout, the callout width is controlled via the dropdownWidth prop. 

See this line from Dropdown for reference https://github.com/microsoft/fluentui/blob/24d2007f2c3889a533f72c7188b89e8a8058414d/packages/react/src/components/Dropdown/Dropdown.base.tsx#L635

You can set the dropdownWidth prop on DropDown to "auto"

Here's an [updated codePen](https://codepen.io/justslone-the-encoder/pen/gOwPvem?editors=0010)

**Reply by gyrevik** [reply]
I'm trying to control the react-northstar dropdown width through a number like 50px but I can't make it take.

**Comment by Rafaelki** [comment]
Thank you very much @JustSlone
